TombEngine/Specific/clock.cpp

```diff
@@ -15,32 +15,32 @@ HighFramerateSynchronizer g_Synchronizer;
 
 void HighFramerateSynchronizer::Init()
 {
-	_controlDelay = 0;
-	_frameTime = 0;
+	_controlDelay = 0.0;
+	_frameTime = 0.0;
+	_locked = false;
 
-	_lastTime.QuadPart = 0;
-	_currentTime.QuadPart = 0;
-	_frequency.QuadPart = 0;
-
-	QueryPerformanceFrequency(&_frequency);
-	QueryPerformanceCounter(&_lastTime);
+	_frequency = SDL_GetPerformanceFrequency();
+	_lastCounter = SDL_GetPerformanceCounter();
+	_currentCounter = _lastCounter;
 }
 
 void HighFramerateSynchronizer::Sync()
 {
-	if (App.ResetClock)
+	if (ResetClock)
 	{
-		App.ResetClock = false;
-		QueryPerformanceCounter(&_lastTime);
-		_currentTime = _lastTime;
-		_controlDelay = 0;
-		_frameTime = 0;
+		ResetClock = false;
+		_lastCounter = SDL_GetPerformanceCounter();
+		_currentCounter = _lastCounter;
+		_controlDelay = 0.0;
+		_frameTime = 0.0;
 	}
 	else
 	{
-		QueryPerformanceCounter(&_currentTime);
-		_frameTime = (_currentTime.QuadPart - _lastTime.QuadPart) * 1000.0 / _frequency.QuadPart;
-		_lastTime = _currentTime;
+		_currentCounter = SDL_GetPerformanceCounter();
+		Uint64 diff = _currentCounter - _lastCounter;
```

Comment:
Let's use data types according to coding conventions, please. Maybe `unsigned long long`?